tox.ini

```diff
@@ -12,6 +12,12 @@ basepython = python3.8
 deps = -r requirements/mini.txt
 commands = pytest tests/minimal_plot.py
 
+[testenv:noplotly]
+description = Test that plotly tests are skipped if plotly is not installed
+basepython = python3.8
+deps = -r requirements/noplotly.txt
+commands = pytest tests/backends/plotly
```

Comment:
So this will not catch any other tests that depend on plotly outside that dir/file?

Reply:
Correct. I don't really want to run all the tests twice?

Reply:
Not if they are slow...?

Reply:
I think it's overkill. If a conda build fails again in the future, we can revisit.
Running the tests will get slower and slower as we add more. In principle, all the plotly stuff should be isolated to this folder.
I understand that a higher-level function in the future might depend on plotly, but we maybe have to rethink the whole repo organisation at that point anyway.